python/tests/table/test_sorting.py

```diff
@@ -68,3 +70,12 @@ def test_sorting_schema():
             ),
         )
     ]
+
+
+def test_sorting_to_string(sort_order: SortOrder):
+    expected = """[
+  identity(19) ASC NULLS_FIRST
```

Comment:
Should there be a comma ending each line since this is in `[...]` as a list?

Reply:
I took inspiration from the Java implementation, and it looks like there is no comma: https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/SortOrder.java#L137-L150